src/Microsoft.Dnx.Watcher/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using Microsoft.Dnx.Watcher.Core;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.Dnx.Watcher
+{
+    public class Program
+    {
+        private const string DnxWatchArgumentSeparator = "--dnxArgs";
```

Comment:
We never camel case command line arguments. Compound words are always `--dnx-args`